resources/troubleshooting-guides/push_notification_troubleshooting.md

```diff
@@ -110,18 +110,18 @@ If SDK privacy status is `optedout`, the push identifier will not be set.
 
    The value in pushIdentifier is the same value that is sent to the ECID service.
 
-   ![Verify SetPushIdentifier event.](../../.gitbook/assets/push_token_to_identity.png)
+![Verify SetPushIdentifier event.](../../.gitbook/assets/push_token_to_identity.png)
```

Comment:
Please move the screenshot over so that it lines up with the start of the text ("The value in `pushIdentifier`...")